src/main/java/net/imagej/legacy/ImageJ2Options.java

```diff
@@ -90,9 +90,10 @@
 	@Parameter(
 		label = "SciJava log level",
 		description = "<html>Log level for SciJava</html",
+		initializer = "initializeLogLevel",
```

Comment:
Please add `persist = false` just for clarity here (even though scijava/scijava-common#236 will probably merge soon).